### PR TITLE
American Modelica Conference Announcement

### PR DESCRIPTION
**Pull request by lvanfretti**
Initial announcement for review.

### CODE REVIEW

File: _2024-01/AmericanModelicaConference.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: "American Modelica Conference 2024"
+author: "Luigi Vanfretti, [North American Modelica Users Group](https://modelica.org/events/american2024/))"
+category: "conference"
+---
+
+We would like to announce the [American Modelica Conference 2024](https://modelica.org/events/american2024/), which will be an in-person conference event, with an option to watch presentations streamed online for a reduced conference admission fee.
+
+The conference will take place at the **University of Connecticut in Storrs** in the [Innovation Partnership Building](https://techpark.uconn.edu/) from **October 14–16, 2024**. It is organized by [NAMUG](https://namug.org/), the North American Modelica Users Group, in cooperation with the [Modelica Association](/association/). Join us in person in Storrs or from the comfort of your own home.
```
Comment (mwetter) [inline _2024-01/AmericanModelicaConference.md:9]: Change North American Modelica Users Group to
North American Modelica Users' Group

File: _2024-01/AmericanModelicaConference.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+The program will cover processes and tools for the modeling of complex physical and cyber-physical systems as applied to a wide range of research and industrial applications.
+
+In addition to paper presentations, the conference features several Modelica tutorials for beginners and advanced users, as well as industrial user presentations, vendor sessions, and an exhibition.  The previous [American Modelica conference in 2022](https://2022.american.conference.modelica.org/) was also streamed, giving many Modelicans in Europe the opportunity to watch presentations on the latest updates on Modelica-related research and innovation.  In this version of the conference, we want to combine the ease of attendance by remote participants from other continents with the stimulating environment of an in person event with lively discussions in the breaks and an in-person conference dinner!
```
Comment (mwetter) [inline _2024-01/AmericanModelicaConference.md:17]: Replace "Europe" by "other continents"

**Review by mwetter — CHANGES_REQUESTED** [review]
@lvanfretti : Please see inline comments (as I don't have write access).

File: _2024-01/AmericanModelicaConference.md (hunk 1 above)
Comment (lvanfretti) [reply _2024-01/AmericanModelicaConference.md]: Applied in [7455e35](https://github.com/modelica/newsletter/pull/287/commits/7455e357f459e9eead09c07e556b81bc00a0b46d)

File: _2024-01/AmericanModelicaConference.md (hunk 2 above)
Comment (lvanfretti) [reply _2024-01/AmericanModelicaConference.md]: Applied in [7455e35](https://github.com/modelica/newsletter/pull/287/commits/7455e357f459e9eead09c07e556b81bc00a0b46d)

**Comment by lvanfretti** [comment]
> @lvanfretti : Please see inline comments (as I don't have write access).

**Comment by lvanfretti** [comment]
@mwetter I've applied your suggestions [7455e35](https://github.com/modelica/newsletter/pull/287/commits/7455e357f459e9eead09c07e556b81bc00a0b46d)

**Review by mwetter — APPROVED** [review]

**Comment by lvanfretti** [comment]
@filip-jezek this is ready for your review.

**Pull request merged**